src/app/calendar/ParcelCalendar.tsx

```diff
@@ -41,18 +43,32 @@ const ParcelCalendar: React.FC<ParcelCalendarProps> = (props) => {
         };
     };
 
-    const colorMap: LocationColorMap = {
-        "Brixton Hill - Methodist Church": getColorText("lightRed"),
-        "Clapham - St Stephens Church": getColorText("lightOrange"),
-        "N&B - Emmanuel Church": getColorText("lightYellow"),
-        "Streatham - Immanuel & St Andrew": getColorText("lightGreen"),
-        "Vauxhall Hope Church": getColorText("darkGreen"),
-        "Waterloo - Oasis": getColorText("lightBlue"),
-        "Waterloo - St George the Martyr": getColorText("darkBlue"),
-        "Waterloo - St Johns": getColorText("lightPurple"),
-        Delivery: getColorText("darkPurple"),
-        default: getColorText("lightBrown"),
-    };
+    // color order deliberately scattered to allow more variety of color in the calendar
```

Comment:
Instead of hardcoding each location with a text, I want it to dynamically load the existing collection centre from the daabase and automatically assign a color for it. The main purpose is such that when new collection centre is made, we don't need developers to edit the code and give the location a new color.

Also, it implies each collection centre doesn't have a fixed color (It might change dynamically if a centre in the middle of the list is deleted). But it is fine, because if not then we should assign a new color to a centre in the database as it is entered, which Tom doesn't seem to prefer because a collection centre doesn't really have the characteristics "color" intrinsically